snippets/distr-tracing-tempo-required-secret-parameters.adoc

```diff
@@ -51,6 +51,16 @@ See link:https://operator.min.io/[MinIO Operator].
 
 `access_key_secret: <s3_access_key_secret>`
 
+|Amazon S3 with Security Token Service (STS)
```

Comment:
@pavolloffay, I can add one link over this line, but I'm having trouble picking one most relevant link. Which one link from the following three would you choose for here?
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sts/index.html
https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html
https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp.html